templates/room.html

```diff
@@ -637,6 +657,15 @@
           'to':to
       }));
     }
+    
+    function sendMessageFromSystem(message,to){
```

Comment:
I will modify the sendMessage function with adding a parameter e.g
sendMessage(message, to, from='SYSTEM')